src/main/java/org/opensearch/jobscheduler/transport/AcquireLockRequest.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.jobscheduler.transport;
+
+import java.io.IOException;
+import java.util.Objects;
+import org.opensearch.action.ActionRequest;
+import org.opensearch.action.ActionRequestValidationException;
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+import org.opensearch.common.xcontent.XContentParser;
+import org.opensearch.common.xcontent.XContentParserUtils;
+
+public class AcquireLockRequest extends ActionRequest {
+
+    private final String jobId;
+
+    private final String jobIndexName;
+
+    private final long lockDurationSeconds;
+
+    public static final String JOB_ID = "job_id";
+    public static final String JOB_INDEX_NAME = "job_index_name";
+    public static final String LOCK_DURATION_SECONDS = "lock_duration_seconds";
+
+    public AcquireLockRequest(StreamInput in) throws IOException {
+        super(in);
+        this.jobId = in.readString();
+        this.jobIndexName = in.readString();
+        this.lockDurationSeconds = in.readLong();
+    }
+
+    public AcquireLockRequest(String jobId, String jobIndexName, long lockDurationSeconds) {
```

Comment:
(Style) Pure preference here, I'd like this constructor first (near the fields) since they are logical pairs, and the StreamInput constructor right next to the `writeTo()` method since they are logical pairs.